docs/reference/commandline/cli.md

```diff
@@ -100,7 +100,15 @@ The property `psFormat` specifies the default format for `docker ps` output.
 When the `--format` flag is not provided with the `docker ps` command,
 Docker's client uses this property. If this property is not set, the client
 falls back to the default table format. For a list of supported formatting
-directives, see the [**Formatting** section in the `docker ps` documentation](ps.md)
+directives, see the
+[**Formatting** section in the `docker ps` documentation](ps.md)
+
+The property `detachKeys` specifies the default key sequence which
+detaches the container. When the `--detach-keys` flag is not provide
+with the `docker attach`, `docker exec`, `docker run` or `docker
+start`, Docker's client uses this property. If this property is not
+set, the client falls back to the default sequence `ctrl-p,ctrl-q`.
+
```

Comment:
Hm, perhaps this becomes confusing, because the default `CTRL-p CTRL-q` (uppercase CTRL, and no comma) must be specified as `ctrl-p,q` in the configuration?

Are other meta-keys, e.g. `ALT` also possible?